src/styles/main.scss

```diff
@@ -1,3 +1,18 @@
+@import "./utils/variables";
+@import "./blocks/cards.scss";
+@import "./blocks/stars.scss";
+@import "./blocks/header.scss";
+@import "./blocks/navBar.scss";
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
 body {
   margin: 0;
 }
```

Comment:
Move these styles to a separate file with general styling to keep your main file clean and clear